Changelogs for Digital Twin Manager Plugins 2.1.2 (#3166)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-2-2-fixed-unresponsive-groups-navigation-in-asset-navigator-within-digital-twin-manager.md

---
date: ""

> title: "Fixed unresponsive "Groups" navigation item in the Digital Twin Manager"

Reviewer: ```suggestion
title: Fixed unresponsive groups navigation item in the Digital Twin Manager
```

> title: "Fixed unresponsive "Groups" navigation in Asset Navigator within Digital Twin Manager"

Reviewer: ```suggestion
title: "Fixed unresponsive "Groups" navigation in asset navigator within Digital Twin Manager"
```

> title: "Fixed unresponsive "Groups" navigation in asset navigator within Digital Twin Manager"

Reviewer: ```suggestion
title: "Fixed unresponsive "Groups" navigation item in the Digital Twin Manager"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1971"
version: "1021.2.2"
---

> Fixed the non-responsive "Groups" navigation item in the Asset Navigator when installed in the Digital Twin Manager application.

Reviewer: ```suggestion
Fixed the unresponsive **Groups** navigation item in the asset navigator when installed in the Digital Twin Manager application.
```

> Fixed the unresponsive **Groups** navigation item in the asset navigator when installed in the Digital Twin Manager application.

Reviewer: ```suggestion
In the Digital Twin Manager, the **Groups** navigation item in the asset navigator was unresponsive. This issue has been fixed. The navigation item now works properly.
```
